staging/src/k8s.io/cloud-provider/ports/ports.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ports
+
+const (
+	// CloudControllerManagerPort is the default port for the cloud controller manager server.
+	// This value may be overridden by a flag at startup.
+	CloudControllerManagerPort = 10258
```

Comment:
Seems like cmd/cloud-controller-manager is the only consumer of this. Wondering if it's simpler to have a local copy of CloudControllerManagerPort there.

Reply:
I vendor it back for maintenance consideration. In this case we have a general place for port and it is much easier to refer to in the future. What would you think?

Reply:
I think vendoring it back into pkg/master/ports/ports.go would be a good idea. I think the advantage of having it here is that cmd/cloud-controller-manager is pretty high in the dependency tree. cloud-provider is much lower, so more things can depend on it if needed. (such as pkg/master)

Reply:
sgtm, do we need a new package for ports though? I think a ports.go file in `k8s.io/cloud-provider` should be fine.

Reply:
I have moved ports.go to k8s.io/cloud-provider